File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> arr = {-1, -1};
        int comp;
        std::unordered_map<int, int> intmap;
        for (int i = 0; i < nums.size(); i++){
            comp = target-nums[i];
            if (intmap.contains(comp)){
                arr[0] = intmap[comp];
                arr[1] = i;
                return arr;
            } else {
                intmap[nums[i]] = i;
            }
        }
        return arr;
    }
};

File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        newa = [0]*(n*2)
        for i in range(n):
            newa[i] = nums[i]
            newa[i+n] = newa[i]
        return newa